File: Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoPaymentMethodUsage.java
package com.braintreepayments.api.venmo;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

/**
 * Usage type for the tokenized Venmo account.
 */
@Retention(RetentionPolicy.SOURCE)
@IntDef({
        VenmoPaymentMethodUsage.SINGLE_USE,
        VenmoPaymentMethodUsage.MULTI_USE
})
public @interface VenmoPaymentMethodUsage {
    int SINGLE_USE = 1;
    int MULTI_USE = 2;
}

File: Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoPaymentMethodUsage.kt
package com.braintreepayments.api.venmo

/**
 * Usage type for the tokenized Venmo account.
 */
enum class VenmoPaymentMethodUsage {

    /**
     * The Venmo payment will be authorized for a one-time payment and cannot be vaulted.
     */
    SINGLE_USE,

    /**
     * The Venmo payment will be authorized for future payments and can be vaulted.
     */
    MULTI_USE
}
